controller_app/views.py

```diff
@@ -11,13 +12,19 @@
 log = logging.getLogger(__name__)
 
 
+def trace_attributes(**attrs):
+    span = trace.get_current_span()
+    span.set_attributes(attrs)
```

Comment:
Worth using tracing.set_span_attributes to handle weird value types passed by accident?

Reply:
yes - I thought about it and then didn't because I mistakenly remembered it doing job-related things